packages/web/src/js/Password.ts

```diff
@@ -4,7 +4,7 @@ import { enableToggleTrigger, SpiritConfig } from './utils';
 import { SpiritElement } from './types';
 
 const NAME = 'password';
-const PASSWORD_ARIA_PRESSED = 'aria-pressed';
```

Comment:
For the record, this is the thing:

> **The `aria-checked` attribute is required when using the `switch` role,** as it represents the current state of the widget that the switch role is applied to.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/switch_role#associated_aria_roles_states_and_properties